examples/const-and-final/const_and_final.bal

```diff
@@ -4,12 +4,18 @@ import ballerina/io;
 const MAX_VALUE = 1000;
```

Comment:
Not introduced by this PR, but do we know why both don't have the type here? I think one can have the type.

Reply:
+1

Reply:
Updated [d250c9f](https://github.com/ballerina-platform/ballerina-distribution/pull/4713/commits/d250c9f661229b921af0f6dc9bea2f0b491595fb)